camel/benchmarks/gaia.py

```diff
@@ -271,6 +274,84 @@ def run(  # type: ignore[override]
 
         return self._generate_summary()
 
+    def run_workforce(  # type: ignore[override]
```

Comment:
maybe we also need to rename `def run` into `def run_chatagent` to follow the same format, same for `BaseBenchmark`